docs/source/get_started/minimal_kedro_project.md

```diff
@@ -0,0 +1,176 @@
+
+### Step 2: Create a New Kedro Project
+Create a new directory for your project:
+```bash
+mkdir minikedro
+```
+
+Navigate into your newly created project directory:
+
+```bash
+cd minikiedro
+```
+
+### Step 3: Create `pyproject.toml`
+Create a new file named `pyproject.toml` in the project directory with the following content:
+
+```toml
+[tool.kedro]
+package_name = "minikedro"
+project_name = "minikedro"
+kedro_init_version = "0.19.9"
+source_dir = "."
+```
+
+At this point, your workingn directory should look like this:
+```bash
+.
+├── pyproject.toml
+```
+
+
+```{note}
+Note we define `source_dir = "."`, usually we keep our source code inside a directory called `src`. For this example, we try to keep the structure minimal so we keep the source code in the root directory
+```
+
+### Step 4: Create `settings.py` and `pipeline_registry.py`
+Next, create a folder named minikedro, which should match the package_name defined in pyproject.toml:
+
+```bash
+mkdir minikedro
+```
+Inside this folder, create two empty files: `settings.py` and `pipeline_registry.py`:
+
+```bash
+touch minikedro/settings.py minikedro/pipeline_registry.py
+```
+
+Now your working directory should look like this:
+```bash
+.
+├── minikedro
+│   ├── pipeline_registry.py
+│   └── settings.py
+└── pyproject.toml
+```
+
+Try running the following command in the terminal:
+```bash
+kedro run
+```
+
+You will encounter an error indicating that pipeline_registry.py is empty:
+```bash
+AttributeError: module 'minikedro.pipeline_registry' has no attribute 'register_pipelines'
+```
+
+### Step 5: Create a Simple Pipeline
+To resolve this issue, add the following code to `pipeline_registry.py`, which defines a simple pipeline to run:
+
+```python
+from kedro.pipeline import pipeline, node
+
+def foo():
+    return "dummy"
+
+def register_pipelines():
+    return {"__default__": pipeline([node(foo, None, "dummy_output")])}
+```
+
+If you attempt to run the pipeline again with `kedro run`, you will see another error:
+```bash
+MissingConfigException: Given configuration path either does not exist or is not a valid directory: /workspace/kedro/minikedro/conf/base
+```
+
+### Step 6: Define the Project Settings
+This error occurs because Kedro expects a configuration folder named `conf`, along with two environments called `base` and `local`.
+
+To fix this, add these two lines into `settings.py`:
+```python
+CONF_SOURCE = "."
+CONFIG_LOADER_ARGS = {"base_env": ".", "default_run_env": "."}
+```
+
+These lines override the default settings so that Kedro knows to look for configurations in the current directory instead of the expected conf folder. For more details, refer to [How to change the setting for a configuration source folder](../configuration/configuration_basics.md#how-to-change-the-setting-for-a-configuration-source-folder) and [Advance Configuration without a full Kedro project](../configuration/advanced_configuration.md#advanced-configuration-without-a-full-kedro-project)
```

Comment:
```suggestion
These lines override the default settings so that Kedro knows to look for configurations in the current directory instead of the expected `conf` folder. For more details, refer to [How to change the setting for a configuration source folder](../configuration/configuration_basics.md#how-to-change-the-setting-for-a-configuration-source-folder) and [Advance Configuration without a full Kedro project](../configuration/advanced_configuration.md#advanced-configuration-without-a-full-kedro-project)
```